src/datasources/targeted-messaging/targeted-messaging.datasource.spec.ts

```diff
@@ -132,6 +133,39 @@ describe('TargetedMessagingDataSource tests', () => {
     });
   });
 
+  describe('getOutreach', () => {
+    it('gets an outreach successfully', async () => {
+      const dto = createOutreachDtoBuilder().build();
+      const created = await target.createOutreach(dto);
+
+      const result = await target.getOutreach(created.id);
+
+      expect(result).toStrictEqual({
+        id: created.id,
+        name: dto.name,
+        startDate: dto.startDate,
+        endDate: dto.endDate,
+        sourceId: dto.sourceId,
+        type: dto.type,
+        teamName: dto.teamName,
+        sourceFile: dto.sourceFile,
+        sourceFileProcessedDate: null,
+        sourceFileChecksum: null,
+        targetAll: false,
```

Comment:
The value is already inside the `DTO` builder, so we can fetch it from the builder object.

Reply:
https://github.com/safe-global/safe-client-gateway/commit/1b72d7acb028af5275397cac1a6b18e63e85e435#diff-ae1f316c87534e60c2f465f11b0301577d8eaf3b6505da5f0bbc49fe31d3fe07L154